### PR TITLE
Update rancher-load-images.sh to support manifest

### PR DESCRIPTION
**Pull request by gitlawr**
Issue: https://github.com/rancher/rancher/issues/22836

## For users who care about Linux only:

The script works exactly like before.

## For users who will use Windows cluster:

- **Step 1:** Users first use `rancher-save-images.ps1` and `rancher-load-images.ps1` to push windows images on Windows 1903 nodes with Docker installed. Recommended Disk Size: 50GB This step pushes the windows tagged images

- **Step 2:** Then users run `./rancher-load-images.sh --registry reg.example.com --image-list ./rancher-images.txt --windows-image-list ./rancher-windows-images.txt` This step pushes the linux tagged images as well as the manifest. 

### Notes
- You have to specify `--windows-image-list ./rancher-windows-images.txt` to indicate you would like to support Windows cluster.
- If you want to use images on 1809 and 1903 instead of 1903 only, they can use `--windows-versions 1809,1903` param

### CODE REVIEW

**Comment by cloudnautique** [comment]
Is there an example for the input files?

**Comment by loganhz** [comment]
example for the input files

https://github.com/rancher/rancher/releases/download/v2.3.0-rc4/rancher-images.txt
https://github.com/rancher/rancher/releases/download/v2.3.0-rc4/rancher-windows-images.txt



We would like to make it in one file in 2.3.x as it requires many changes.
Here is the issue for tracking it https://github.com/rancher/rancher/issues/22865

File: pkg/image/export/main.go [hunk 1]
```diff
@@ -224,14 +224,31 @@ const (
...
 images="rancher-images.tar.gz"
 list="rancher-images.txt"
+windows_image_list=""
+windows_versions="1903"
 usage () {
     echo "USAGE: $0 [--images rancher-images.tar.gz] --registry my.registry.com:5000"
     echo "  [-l|--image-list path] text file with list of images. 1 per line."
     echo "  [-i|--images path] tar.gz generated by docker save."
     echo "  [-r|--registry registry:port] target private registry:port."
+    echo "  [--windows-image-list path] text file with list of images used in Windows. Windows image mirroring is skipped when it is empty"
```
Comment (cjellick) [inline pkg/image/export/main.go]: "when it is empty" is awkward english. "when this is empty" would be better
can be changed in a separate PR
